config/puma.rb

```diff
@@ -1,2 +1,4 @@
 port ENV['PORT'] || 3000
 environment ENV['RAILS_ENV'] || 'development'
+workers 1
+threads 0, 16
```

Comment:
So it looks like the [0, 16] `threads` is the default.  But `workers` defaults to 0... so how is workers = 1 any different?  It switches to clustered mode, but with a single forked thread ... doesn't that amount to the same thing?

Reply:
no, it's 2 processes, a master and a worker, instead of everything in 1 process:
https://github.com/puma/puma/issues/1364

Reply:
which is to say, it's different from the default single mode, but I have no idea how differently this will perform - seems like the same draw on db connections, but with a different master thread, maybe easier to interrupt and manage? idk